cmake/CreateGrfCommand.cmake

```diff
@@ -42,9 +43,14 @@ function(create_grf_command)
             COMMENT "Generating ${GRF_SOURCE_FOLDER_NAME}.grf"
     )
 
-    # For conviance, if you want to only test building the GRF
+    # For convenience, if you want to only test building the GRF
     add_custom_target(${GRF_SOURCE_FOLDER_NAME}.grf
             DEPENDS
             ${GRF_BINARY_FILE}
     )
+
+    add_custom_target(${GRF_SOURCE_FOLDER_NAME}.grf.hash
+            DEPENDS
+            ${GRF_BINARY_FILE}
+    )
```

Comment:
This target will do the same as `${GRF_SOURCE_FOLDER_NAME}.grf` target, I don't think it's needed to add it.